### DISCUSSION TITLE
DMCA for a fork

### DISCUSSION THREAD

**Discussion by PERCE-NEIGE** [community/community#136339] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hi is there the right place to contact the support ?
I have forked a software only for training, and this software has been targeted by a DMCA complaint. So my whole repository has been banned. But I only forked something I've found on Github, how I was supposed to know the software was not legal ?

Please tell me how to reactivate it.

**Comment by srijon57** [comment]
Hi PERCE-NEIGE my friend,

Thank you for reaching out. I understand your concern regarding the DMCA complaint and the subsequent ban of your repository.

Firstly, it's important to note that forking a repository on GitHub does not automatically mean you are aware of its legal status. However, GitHub takes DMCA complaints seriously to protect intellectual property rights.

Here are the steps you can take to address this issue:

#Understand the DMCA Complaint: Review the DMCA notice you received. It should provide details about the alleged infringement.

#Contact GitHub Support: The best way to resolve this issue is to contact GitHub Support directly. You can do this by visiting the [GitHub Support page](https://support.github.com/contact) and submitting a request.

#File a Counter-Notice: If you believe the DMCA complaint was filed in error or that the content is not infringing, you can file a counter-notice. GitHub provides a [guide on how to file a DMCA counter-notice](https://docs.github.com/en/github/site-policy/dmca-takedown-policy#b-filing-a-dmca-counter-notice).

#Provide Details: When contacting GitHub Support, make sure to provide them with the following information:

Your GitHub username.
 The URL of the banned repository.
A brief explanation of the situation, including the fact that you forked the repository for training purposes and were unaware of any legal issues.
Wait for a Response: GitHub Support will review your case and provide further instructions or resolve the issue if appropriate.

In the future, it's a good practice to check the licensing and legal status of any software you plan to fork or use, especially if it's for more than personal training purposes.
